indico/modules/events/abstracts/clone.py

```diff
@@ -21,7 +21,7 @@
 class AbstractSettingsCloner(EventCloner):
     name = 'abstracts_settings'
     friendly_name = _('Call for Abstracts (settings, email templates, review questions)')
-    requires = {'contribution_types', 'tracks'}
+    requires = {'contribution_types', 'tracks', 'contribution_fields'}
```

Comment:
```suggestion
    requires = {'contribution_fields', 'contribution_types', 'tracks'}
```

just a bit of OCDing about the order here :P

Reply:
Done